pipeline-model-definition/src/main/resources/org/jenkinsci/plugins/pipeline/modeldefinition/PropertiesToMapTranslator.groovy

```diff
@@ -52,16 +57,18 @@ public class PropertiesToMapTranslator implements MethodMissingWrapper, Serializ
             argValue = args[0]
         }
 
-        if (resolveCredentials && s == "credentials") {
-            if (args.length == 1) {
-                String id = "${args[0]}"
-
-                RunWrapper currentBuild = script.getProperty("currentBuild")
+        if (resolveEnvironmentContributors) {
+            def retVal
+            if (argValue != null) {
```

Comment:
Yepp, I fixed it in the other PR, there is now a util method that does about the same thing as `script."${name}"(args)` would do but you can search for a more specific `Describable` descendant. :-D